script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -730,32 +870,36 @@ As long as you don't try to run, the Dead won't lay a hand on you.
 [sha:2d67afdd02d9a49be1c49764b0657901758f8f75]{
 -- Page 158, Offset 35700.
 -- 次は指一本じゃなくて、ちゃんと平手打ちで真っ二つにしてあげる。この舞踏場の中で、ボロボロになるまでわたしと楽しみましょう？」
-Instead of only using one finger, this time I'll smack you right in half. Now, shall we enjoy ourselves within this ballroom until you've been torn to shreds?"
+I won't just use one finger this time――I'll smack you clean in half.
+ Now, would you like to dance yourself ragged with me in this ballroom?"
```

Comment:
Latter sentence is NOT A QUESTION (it's rhetorical)) NOEL DEMANDS DANCIES
Something like this?
>Now, why don't you dance yourself ragged here, together with me in this ballroom?